packages/html/src/globals.ts

```diff
@@ -0,0 +1,31 @@
+import { bigtestGlobals } from "@bigtest/globals";
+
+interface Globals {
+  documentResolver: () => Document;
+}
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace, @typescript-eslint/prefer-namespace-keyword
+  module globalThis {
+    // eslint-disable-next-line prefer-let/prefer-let, no-var
+    var __interactors: Globals;
+  }
+}
+
+if (!globalThis.__interactors) {
+  globalThis.__interactors = {
+    documentResolver: () => bigtestGlobals.document,
+  };
+} else {
+  setDocumentResolver(() => bigtestGlobals.document);
+}
```

Comment:
I don't think we want this `else`branch here because this will set the document resolver every time this module is evaluated from different contexts.